### PR TITLE
Feature: Update user settings (UI), removed footer temporarily due to style bugs.

### PR DESCRIPTION
**Pull request by andytubeee**
If you're opening a pull request for this repo, then do answer a few questions below, before submitting it!

Does your PR solve any issues opened or raised in this repo or from the project's bug section?
  - [ ] Yes
  - [X] No
  
If you've mentioned **Yes** above, then mention the Issue no here (with #), else ignore it:

**Issue No**: 

Is it a bug fix, security fix, feature update, UI Update or other?
  - [ ] Bug fix (related to Firebase, backend etc)
  - [X] Feature Update
  - [ ] Page addition
  - [X] UI Fix
  - [ ] Other
  
If you've mentioned Other, then state your answer here (like what actually is and that):

If page addition is marked, mention the page name here:

Have you notified this PR of the issue? [OPTIONAL] (so that we can check in case of issue fix)
  - [ ] Yes
  - [X] No

### CODE REVIEW

**Comment by alphaX86** [comment]
Next time, put X and not * to mark the list items in the PR template @andytubeee

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```
Comment (alphaX86) [inline src/App.js:1]: Is this allowed normally?

Comment (GedionT) [reply src/App.js:1]: The second line is enough. Remove the first line.

Comment (alphaX86) [reply src/App.js:1]: Well I think he removed

**Review by alphaX86 — APPROVED** [review]
Changes approved with co-reviewer

**Comment by alphaX86** [comment]
Firebase CD is failing

**Pull request merged**